!Command
command:
- sample2markers.py
positional: []
named: []
parent:
subcommands: []
usage: []
help_flag:
usage_flag:
version_flag:
help_text: "Traceback (most recent call last):\n  File \"/usr/local/bin/sample2markers.py\"\
  , line 53, in <module>\n    request.urlretrieve('https://bitbucket.org/biobakery/metaphlan2/downloads/bcftools',\
  \ os.path.join(os.path.dirname(os.path.realpath(__file__)),'bcftools_0.1.19'))\n\
  \  File \"/usr/local/lib/python3.7/urllib/request.py\", line 247, in urlretrieve\n\
  \    with contextlib.closing(urlopen(url, data)) as fp:\n  File \"/usr/local/lib/python3.7/urllib/request.py\"\
  , line 222, in urlopen\n    return opener.open(url, data, timeout)\n  File \"/usr/local/lib/python3.7/urllib/request.py\"\
  , line 531, in open\n    response = meth(req, response)\n  File \"/usr/local/lib/python3.7/urllib/request.py\"\
  , line 641, in http_response\n    'http', request, response, code, msg, hdrs)\n\
  \  File \"/usr/local/lib/python3.7/urllib/request.py\", line 569, in error\n   \
  \ return self._call_chain(*args)\n  File \"/usr/local/lib/python3.7/urllib/request.py\"\
  , line 503, in _call_chain\n    result = func(*args)\n  File \"/usr/local/lib/python3.7/urllib/request.py\"\
  , line 649, in http_error_default\n    raise HTTPError(req.full_url, code, msg,\
  \ hdrs, fp)\nurllib.error.HTTPError: HTTP Error 403: Forbidden\n"
generated_using:
- --help
